src/engraving/tests/tst_copypaste.cpp

```diff
@@ -82,12 +83,21 @@ private slots:
 
     void copypastestaff50() { copypastestaff("50"); }         // staff & slurs
 
-    void copyPastePartial();
-
-    void copyPasteTuplet01() { copypastetuplet("01"); }
-    void copyPasteTuplet02() { copypastetuplet("02"); }
-
-    //void copyPasteTremolo01() { copypastetremolo(); }
+    void copypastePartial();
+
+    void copypasteTuplet01() { copypastetuplet("01"); }
+    void copypasteTuplet02() { copypastetuplet("02"); }
+    void copypasteQtrNoteOntoWholeRest() { copypastenote("01"); }
+    void copypasteQtrNoteOntoWholeNote() { copypastenote("02"); }
+    void copypasteQtrNoteOntoQtrRest() { copypastenote("03"); }
+    void copypasteQtrNoteOntoQtrNote() { copypastenote("04"); }
+    void copypasteWholeNoteOntoQtrNote() { copypastenote("05"); }
+    void copypasteWholeNoteOntoQtrRest() { copypastenote("06"); }
+    void copypasteQtrNoteOntoTriplet() { copypastenote("07"); }
+    void copypasteWholeNoteOntoTriplet() { copypastenote("08"); }
+    void copypasteQtrNoteIntoChord() { copypastenote("09"); }
```

Comment:
pretty minimal set of tests, there's quite a few more edge cases I'd like to have tests for, but for some them the "desired" behaviour isn't necessarily what we're getting now, I guess at least having a test to ensure it doesn't crash would be good!